pkg/cmd/server/kubernetes/node.go

```diff
@@ -192,77 +191,77 @@ func (c *NodeConfig) RunSDN() {
 
 // RunProxy starts the proxy
 func (c *NodeConfig) RunProxy() {
```

Comment:
most of these changes are reordering/renaming to match the upstream proxy start code, or pulling out config bits that are now computed in buildKubeProxyConfig